support-frontend/app/admin/settings/Switches.scala

```diff
@@ -33,6 +33,7 @@ case class CampaignSwitches(
     enableContributionsCampaign: Option[SwitchState],
     forceContributionsCampaign: Option[SwitchState],
     usEoy2024: Option[SwitchState] = None,
+    ausTicker2024: Option[SwitchState] = None,
```

Comment:
only a name, but wouldn't `ausEoy2024` be more accurate?

Reply:
ok , I will change that